src/datanode/src/heartbeat.rs

```diff
@@ -103,10 +127,35 @@ impl HeartbeatTask {
         let meta_client = self.meta_client.clone();
 
         let catalog_manager_clone = self.catalog_manager.clone();
-        let mut tx = Self::create_streams(&meta_client, running.clone()).await?;
+        let mailbox = self.mailbox.clone();
+        let handler_executor = self.heartbeat_response_handler_exector.clone();
+        let mut tx = Self::create_streams(
+            &meta_client,
+            running.clone(),
+            handler_executor.clone(),
+            mailbox.clone(),
+        )
+        .await?;
+
         common_runtime::spawn_bg(async move {
             while running.load(Ordering::Acquire) {
                 let (region_num, region_stats) = datanode_stat(&catalog_manager_clone).await;
+                let mut mailbox_guard = mailbox.lock().await;
```

Comment:
When metasrv sends a request through mailbox, it may hope to receive the fastest response. It seems that we are responding to the mailbox using the timed heartbeat reply mechanism. I wonder if we can respond to the mailbox immediately.